agent/api/task/task.go

```diff
@@ -250,6 +252,18 @@ func (task *Task) PostUnmarshalTask(cfg *config.Config,
 		}
 	}
 
+	err := task.initializeContainerOrderingForVolumes()
```

Comment:
Have we already validated the links / volumes by now?

Reply:
No I think. The links/volumes are validated just before before creating the container, while adding the hostconfig.